Sources/BuildSystemIntegration/BuildSystemManager.swift

```diff
@@ -382,7 +382,6 @@ extension BuildSystemManager: BuildSystemDelegate {
 }
 
 extension BuildSystemManager {
-  // FIXME: Consider debouncing/limiting this, seems to trigger often during a build.
```

Comment:
I guess this hasn't proved to be much of an issue?

Reply:
I am not aware of any issue caused by not debouncing this in the last 5 years…